core-it-suite/src/test/java/org/apache/maven/it/MavenITmng4720DependencyManagementExclusionMergeTest.java

```diff
@@ -39,11 +40,13 @@ public MavenITmng4720DependencyManagementExclusionMergeTest() {
     /**
```

Comment:
Should I update the `super()` above? Maybe keep the old test for `[2.0.6,4.0.0)` and move these changes to a new test for `[4.0.0,)`?

I don’t know how this test suite is used for previous releases…

Reply:
It's not really used anymore. At least, we branched for Maven 3.x, not sure how we'll handle things for 4.0, 4.1, etc..., but I was actually considering merging ITs into maven to ease.  I don't really see any good reason not to do that.